frontend/src/scenes/groups/groupLogic.ts

```diff
@@ -39,6 +40,13 @@ export const groupLogic = kea<groupLogicType>({
                 setGroup: (_, { groupKey }) => groupKey,
             },
         ],
+        activeCardTab: [
+            'properties' as 'properties' | 'related',
```

Comment:
worth saving this type in `groupLogic` as `CardTabTypes` ?

Reply:
/shrug, it is a two-of :D

Reply:
and used in 3 places across 2 files with the same strings repeated as type definitions 😅 . No strong feelings though, hence the approval!